ngraph_onnx/onnx_importer/ops_bridge.py

```diff
@@ -745,9 +745,9 @@ def BatchNormalization(onnx_node, ng_inputs):  # type: (NodeWrapper, List[Ngraph
 
 def Shape(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> NgraphNode
     """Return input shape."""
-    return ng.constant(ng_inputs[0].shape, dtype=int)
+    return ng.constant(ng_inputs[0].shape, dtype=np.int64)
```

Comment:
Please explain the reason for this change (perhaps in a comment).
Can we avoid hard-coding the type?

Reply:
It is an arbitrary type required by actual ONNX master tests.
Without this change we will fail ONNX tests in the near future.